### DISCUSSION TITLE
Can I already use nu Voigt?

### DISCUSSION THREAD

**Discussion by nicocopez** [h2020charisma/ramanchada2#69] [Q&A]
Hi,
not sure if I understood correctly, but it is nu voigt already implemented in rc2? How could I use it?
Is there something like 
`fitres = spe.fit_peak_multimodel(profile='NuVoigt', candidates=peak_candidates, no_fit=False)` ?
Thank you,
Nico

**Comment by georgievgeorgi** [comment]
AFAIK it is almost completely implemented in `FitFuncVoigt` branch. I am not sure however that it is included in `fit_peak_multimodel` function. I didn't have chance to go through all proposed changes yet.
